R/add_predicted_samples.R

```diff
@@ -161,9 +181,15 @@ predicted_samples.brmsfit = function(model, newdata, var = "pred", ..., n = NULL
   if (!requireNamespace("brms", quietly = TRUE)) {
     stop("The `brms` package is needed for `predicted_samples` to support `brmsfit` objects.", call. = FALSE)
   }
-
+  
+  if (hasArg(nsamples)) { # See https://github.com/mjskay/tidybayes/issues/70
+    stop("`tidybayes` has standardized some arguments for [add_]predicted_samples",
+         " Please use `n` rather than `nsamples`. See the documentation for",
+         " more details.")
+  }
+  
   fitted_predicted_samples_brmsfit_(predict, model, newdata, var, ...,
-    nsamples = n, re_formula = re_formula
+                                    nsamples = n, re_formula = re_formula
```

Comment:
I'm not a big fan of indent-to-match-preceding-brace style, it creates whitespace dependencies between lines (requiring maintenance) and wastes horizontal space. If you prefer lining up the args here, I would prefer this style:

```r
  fitted_predicted_samples_brmsfit_(
    predict, model, newdata, var, ...,
    nsamples = n, re_formula = re_formula
  )

Reply:
The current way works for me:
```
  fitted_predicted_samples_brmsfit_(predict, model, newdata, var, ...,
    nsamples = n, re_formula = re_formula
  )
```
I didn't even intend to change it, thanks for pointing it out.